task_sdk/tests/defintions/test_dag.py

```diff
@@ -417,3 +417,9 @@ def noop_pipeline(value): ...
         # Test that if arg is not passed it raises a type error as expected.
         with pytest.raises(TypeError):
             noop_pipeline()
+
+    def test_create_dag_while_active_context(self):
+        """Test that we can safely create a DAG whilst a DAG is activated via ``with dag1:``."""
+        with DAG(dag_id="simple_dag"):
+            DAG(dag_id="dag2")
+            # No asserts needed, it just needs to not fail
```

Comment:
```suggestion
            assert True
```

Reply:
Maybe :D ?

Reply:
🤷🏻 We have other tests that rely on things not throwing an exception, but nothing else I can see that does `assert True`

Reply:
assert True is redundant :D

Reply:
Why? At some point we might add a rule "flag tests without asserts" if that will be added in Ruff - which might be actually a useful suggestion for a new rule for them.....

Reply:
I value consistency more tbh.

This did make me discover a couple of other yoda conditions we have in test. `assert True is <some_expr>` style of things.

Reply:
sure . That works too